python/pyspark/streaming/tests.py

```diff
@@ -61,9 +61,12 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.sc.stop()
         # Clean up in the JVM just in case there has been some issues in Python API
-        jSparkContextOption = SparkContext._jvm.SparkContext.get()
-        if jSparkContextOption.nonEmpty():
-            jSparkContextOption.get().stop()
+        try:
```

Comment:
Is this `try` block necessary? If so, would you mind logging the exception in the `except` branch, just so that it's not swallowed silently?